api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/StandardModelSelector.java

```diff
@@ -24,4 +24,9 @@ public OntModel getModel(HttpServletRequest request, ServletContext context) {
 
     public static final ModelSelector selector = new StandardModelSelector();
 
+	@Override
+	public String getDefaultGraphUri() {
+		return ABOX_UNION;
```

Comment:
I found some errors in forms while using ABOX_UNION. 
I haven't found any problems with ABOX_ASSERTIONS as default graph.
@brianjlowe Do you think it is ok to use ABOX_ASSERTIONS as a default graph?